employee-management/webapp/Component.js

```diff
@@ -1,9 +1,12 @@
 sap.ui.define([
-        "sap/ui/core/UIComponent",
-        "sap/ui/Device",
-        "ca/deloitte/hr/employeemanagement/model/models"
-    ],
-    function (UIComponent, Device, models) {
+    "sap/ui/core/UIComponent",
+    "sap/ui/Device",
+    "ca/deloitte/hr/employeemanagement/model/models",
+    'sap/f/library',
+    'sap/ui/model/json/JSONModel'
+
+],
+    function (UIComponent, Device, models, fioriLibrary, JSONModel) {
```

Comment:
It seems you are not using JSONModel in the code, it not, please remove it from here and line 6.